src/UI/processesstorage.cpp

```diff
@@ -0,0 +1,89 @@
+#include "pch.h"
+#include "processesstorage.h"
+
+ProcessesStorage::ProcessesStorage(){};
+
+ProcessesStorage::ProcessesStorage(std::initializer_list<ProcessInfo> list){
+    for(auto& process : list){
+    processes.emplace(std::make_pair(process.Pid, process));
+}
+};
+
+ProcessesStorage::~ProcessesStorage(){}
+
+void ProcessesStorage::add(const ProcessInfo&& processInfo){
+    const DWORD Pid = processInfo.Pid;
+    if(processes.find(Pid) != processes.end()){
+        throw std::invalid_argument("Process with given PID already exist in storage !");
+    }
+    processes.emplace(std::make_pair(Pid,std::move(processInfo)));
```

Comment:
`std::move` will not work with `const rvalue`, remove `const` modifier to make this move effective.